### PR TITLE
Adding Hotglue app

### PR DESCRIPTION
**Pull request by k0a1a**
Hi there, 

I packaged by own project to be installed on YunoHost.

Thank you!

### CODE REVIEW

**Comment by tituspijean** [comment]
Welcome and thank you for the packaging!

Would you consider transferring the repository to the YunoHost-Apps organization if we invite you to it? It provides access to our CI servers to help your packaging and automagic update of the sources whenever the upstream app has a new release.

(it's totally fine refusing to do so too 😄)

**Comment by k0a1a** [comment]
Hi,

thanks for accepting my contribution! YTes, I'd love to transfer the repo to YunoHost-Apps but it's seems I need to be added to the org first?

**Comment by OniriCorpe** [comment]
invite sent ^w^

File: apps.toml [hunk 1]
```diff
@@ -1601,6 +1601,13 @@ state = "working"
 subtags = [ "email" ]
 url = "https://github.com/YunoHost-Apps/horde_ynh"
 
+[hotglue]
+added_date = 1722956791 # 2024/08/06
+category = "publishing"
+potential_alternative_to = [ "Wordpress" ]
+state = "working"
+url = "https://github.com/k0a1a/hotglue_ynh"
```
Comment (ericgaspar) [inline apps.toml:1609]:
```suggestion
url = "https://github.com/YunoHost-Apps/hotglue_ynh"
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**